litmus/sched_gsn_edf.c

```diff
@@ -474,8 +476,20 @@ static struct task_struct* gsnedf_schedule(struct task_struct * prev)
 	 * this. Don't do a job completion if we block (can't have timers running
 	 * for blocked jobs).
 	 */
-	if (!np && (out_of_time || sleep))
-		curr_job_completion(!sleep);
+	if (!np && (sleep || out_of_time)){
```

Comment:
Nit: Write the second half as `(out_of_time || sleep)` to minimize the difference from the original version.